rust/src/lang/english.rs

```diff
@@ -24,6 +24,7 @@ lazy_static! {
         (TranslationIds::EmergencyModePushNotificationBody, "is in an EMERGENCY! Please CONFIRM that they are okay!!"),
         (TranslationIds::NormalModePushNotificationBody, "is no longer in an emergency."),
         (TranslationIds::InvalidHistoricalLocationStartTime, "It is not possible to obtain the location from more than a week ago."),
-        (TranslationIds::PushNotificationActionView, "Go to app")
+        (TranslationIds::PushNotificationActionView, "Go to app"),
+        (TranslationIds::CannotUseOwnInvitation, "You are dealing with an invitation that you created.\nArmore is designed for you to share your location with the people you love.\nTo achieve this, you must send the invitation (link) to the person you want to follow you.\nIf you have any questions, go to the profile section and ask us anything by email or discord.")
```

Comment:
You are trying to accept an invitation that you created

Reply:
For now I'll go for "You are using an invitation that you created" so I can use the same for accept and reject, what do you think?

Reply:
@darioalessandro I've changed it to: "You are using an invitation that you've created", how about that? :D

Reply:
@JasterV I think that it should be more specific, something like: "You are using and invitation that you created. Send this link to a loved one from them to befriend you...."